downstream/modules/hub/proc-delete-namespace.adoc

```diff
@@ -12,12 +12,18 @@ You must first ensure that the namespace you want to delete does not contain a c
 * You have *Delete namespace* permissions.
 
 .Procedure
-//[ddacosta] For 2.5 this will be Log in to Ansible Automation Platform and select Automation Content. Automation hub opens in a new tab. From the navigation ...
+
 . Log in to your {PlatformNameShort}.
 . From the navigation panel, select {MenuACNamespaces}.
 . Click the namespace to be deleted.
 . Click the btn:[More Actions] icon *{MoreActionsIcon}*, then click btn:[Delete namespace].
 +
-NOTE: If the btn:[Delete namespace] button is disabled, the namespace contains a collection with dependencies. Review the collections in this namespace, and delete any dependencies. See link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/getting_started_with_automation_hub/uploading-content-hub#delete-collection[Deleting a collection on automation hub] for information.
+[NOTE]
+
```

Comment:
I don't know if this will cause an issue but there's an extra line in this note. Line 22.

Reply:
updated. thanks for catching it!